mode/examples/Topics/Motion/CubesWithinCube/CubesWithinCube.pyde

```diff
@@ -6,22 +6,21 @@ Collision detection against all outer cube's surfaces.
 """
 from cube import Cube
 
-halfWidth = None
-halfHeight = None
 
 Bounds = 300
 # 20 little internal cubes
 cubes = []
-for i in range(20):
```

Comment:
Mmm... I see your point. I kind of took the long way around, but I was thinking pretty much the same thing ("Why on earth are we even using `setup` et. al....? Oh... Java") earlier today.